projects/core/src/lib/components/print/print/print.component.css

```diff
@@ -2,3 +2,16 @@
   display: block;
   padding: 14px 14px 0 14px;
 }
+
+::ng-deep .lowerHeight .mat-button-toggle-label-content {
```

Comment:
In case ::ng-deep is needed, we should prefix the selector with something component specific. In this case I think it could easily be changed to:

.lowerHeight ::ng-deep .mat-button-toggle-label-content

This way the CSS is still scoped to the component.